tests/imageupload_test.go

```diff
@@ -77,19 +76,19 @@ var _ = Describe("[Serial]ImageUpload", func() {
 
 	validateDataVolume := func(targetName string) {
 		By("Get DataVolume")
-		_, err := virtClient.CdiClient().CdiV1alpha1().DataVolumes(namespace).Get(targetName, metav1.GetOptions{})
+		_, err := virtClient.CdiClient().CdiV1alpha1().DataVolumes(tests.NamespaceTestDefault).Get(targetName, metav1.GetOptions{})
```

Comment:
Regarding the "Reference the default namespace directly " commit: Accessing `tests.NamespaceTestDefault` seems unsafe as it may change now. It may be safer to protect it through a function getter that validates it is "safe".
I guess in any case, we should not allow it to be referenced in a pre-populated state.

Reply:
> I guess in any case, we should not allow it to be referenced in a pre-populated state.

Yes. Further the parallelization model of ginkgo goes like this: It starts one binary per ginkgo node. So at least from that side there is no special protection necessary.